src/components/Tabs/index.tsx

```diff
@@ -43,6 +44,7 @@ const Tabs = ({ tabs, onTabChange, activeTab }: TabsProps): JSX.Element => {
       alignItems: 'center',
     },
     '& .MuiTab-iconWrapper': {
+      fill: '#7F775B',
```

Comment:
I think we want to avoid hard-coded hex values. I believe we want `theme.palette.TwClrIcnSecondary as string` here.

Reply:
I'll update this to use the color token, but heads up that the hex value is also incorrect for this color token.

Reply:
I'm seeing `TwClrIcnSecondary`/`tw-clr-icn-secondary` mapping to `#7F775B` on Storybook. I think that's correct?

Reply:
You're correct, I'm not sure what I thought I saw 😅